packages/interaction/src/InteractionManager.js

```diff
@@ -988,9 +1010,11 @@ export default class InteractionManager extends EventEmitter
      *  interactionEvent, displayObject and hit will be passed to the function
      * @param {boolean} [hitTest] - this indicates if the objects inside should be hit test against the point
      * @param {boolean} [interactive] - Whether the displayObject is interactive
+     * @param {boolean} [noDelayed] - Whether to process delayed events before returning. This is
+     *  used to avoid processing them too early during recursive calls.
      * @return {boolean} returns true if the displayObject hit the point
      */
-    processInteractive(interactionEvent, displayObject, func, hitTest, interactive)
+    processInteractive(interactionEvent, displayObject, func, hitTest, interactive, noDelayed)
```

Comment:
Would prefer `delayed` here instead of the opt-in negative. This creates a double-negative with `!noDelayed` which is confusing and less readable. 
```suggestion
    processInteractive(interactionEvent, displayObject, func, hitTest, interactive, delayed = true)
```

Reply:
I'm personally no fan of default positives in function parameters, because if you use them
these become different calls:

```js
this.processInteractive(a, b, c, d, c, null); // delayed = null (false-ey)
this.processInteractive(a, b, c, d, c);       // delayed = true
````

You would either have to add code changing everything but ``false`` back to ``true``
or let the user deal with that.

Either would be worse than just making it a default false IMHO.

Reply:
`delayed === false` instead of `!delayed` false-y coercion  ;)

Reply:
What about naming the parameter “skipDelayed” to avoid the double negative?